### PR TITLE
Story(CCLS-2194) return checkpoint data from get assessment/assessments

### PR DESCRIPTION
**Pull request by PhilDigitalJustice**
Also includes:

CCLS-2100 - delete checkpoint endpoint

### CODE REVIEW

File: assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/OpaCheckpoint.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * Represents an Oracle Intelligence Advisor Checkpoint.
+ */
+@Entity
+@Table(name = "XXCCMS_OPA_CHECKPOINT")
+@Data
+public class OpaCheckpoint {
+
+  @Id
+  @Column(name = "RESUME_ID")
+  private long resumeId;
```
Comment (porritta) [inline assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/OpaCheckpoint.java:24]: Should be Long. And could we just call it 'id'?

**Review by porritta — CHANGES_REQUESTED** [review]
review complete

File: assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/OpaCheckpoint.java (hunk 1 above)
Comment (PhilDigitalJustice) [reply assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/OpaCheckpoint.java]: Amended

**Review by porritta — APPROVED** [review]
LGTM

**Pull request merged**